airflow/contrib/hooks/neo4j_hook.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This hook provides minimal thin wrapper around the neo4j python library to provide query execution"""
+from neo4j import BoltStatementResult, Driver, GraphDatabase, Session
+
+from airflow.hooks.base_hook import BaseHook
+
+
+class Neo4JHook(BaseHook):
```

Comment:
Would it make sense to inherit from DbApiHook ?

Reply:
I considered this, however cypher queries are not really sql.
Also I was not using a Neo4j connection type and wanted to avoid the complexity of using a ConnectorProtocol or getting into SQL Alchemy.
Whilst Neo4j can be treated like a database, it does not feel like a natural fit for DbApiHook.